x-pack/plugins/canvas/public/expression_types/arg_types/container_style/extended_template.js

```diff
@@ -7,13 +7,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { EuiSpacer, EuiTitle } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
 import { BorderForm } from './border_form';
 import { AppearanceForm } from './appearance_form';
 
 export const ExtendedTemplate = ({ getArgValue, setArgValue, workpad }) => (
   <div>
     <EuiTitle size="xxxs" textTransform="uppercase">
-      <h6>Appearance</h6>
+      <h6>
+        <FormattedMessage
+          id="xpack.canvas.expressionTypes.containerStyle.appearanceTitle"
```

Comment:
xpack.canvas.expressionTypes.containerStyle => xpack.canvas.containerStyle for all ids in this file